enum4linux: add page (#3810)
tldr-pages/tldr: pages/linux/enum4linux.md

# enum4linux

> > Enum4linux is a tool for enumerating information from Windows and Samba systems.

Reviewer: Perhaps this will read better?
```suggestion
> Tool for enumerating Windows and Samba information from remote systems.
```

> > It attempts to offer similar functionality to enum.exe formerly available from www.bindview.com.

Reviewer 2: Should we have `enum.exe` inside backticks here? I've seen multiple forms of writing it here on tldr.

Reviewer 3: Probably yes. The problem is that this has never been standardized, i.e. codified into the style guide.

> 

Reviewer: Is there a homepage link we can include here?

Author: Yes there is here: https://github.com/portcullislabs/enum4linux

- Try to enumerate using all methods:

> `enum4linux -a remote_host`

Reviewer: Token syntax?
```suggestion
`enum4linux -a {{remote_host}}`
```

> - Enumerate using login:

Reviewer: Perhaps this will read better?
```suggestion
- Enumerate using given login credentials:
```

> `enum4liux -u user_name -p password remote_host`

Reviewer: Does it prompt for a password if you omit it? That would be more secure.

Also, token syntax?
```suggestion
`enum4liux -u {{user_name}} -p {{password}} {{remote_host}}`
```

Finally, are there any long forms of these options? We tend to prefer those (same goes for the examples further down.

> - Enumerate host for usernames:

Reviewer 4: List usernames from a given host.

Reviewer 3: @agnivade I edited your comment to make it a suggestion. I hope that's okay. :p

> `enum4liux -U remote_host`

Reviewer: Token syntax?
```suggestion
`enum4liux -U {{remote_host}}`
```

- List shares:

> `enum4liux -S remote_host`

Reviewer: Token syntax?
```suggestion
`enum4liux -S {{remote_host}}`
```

- Get OS information:

> `enum4liux -o remote_host`

Reviewer: Token syntax?
```suggestion
`enum4liux -o {{remote_host}}`
```

# enum4linux

> Enum4linux is a tool for enumerating information from Windows and Samba systems.
> It attempts to offer similar functionality to enum.exe formerly available from www.bindview.com.

- Try to enumerate using all methods:

`enum4linux -a remote_host`

- Enumerate using login:

`enum4liux -u user_name -p password remote_host`

- Enumerate host for usernames:

`enum4liux -U remote_host`

- List shares:

`enum4liux -S remote_host`

> 

Reviewer 3: ```suggestion
```
